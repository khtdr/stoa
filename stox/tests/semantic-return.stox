#!/usr/bin/env stox

{ // should't be allowed
  return;
}
