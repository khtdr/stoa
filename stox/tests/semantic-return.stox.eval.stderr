Parse error in ./semantic-return.stox at line,col 4,3
4.   return;
     â†‘
No return from top-level allowed
